gwswlib/hydx.py

```diff
@@ -463,12 +552,14 @@ class Hydx:
         },
         "Kunstwerk1.csv": {"hydx_class": Structure, "collection_name": "structures"},
         "Verbinding1.csv": {"hydx_class": Connection, "collection_name": "connections"},
+        "Profiel1.csv": {"hydx_class": Profile, "collection_name": "profiles"},
```

Comment:
Tip: kijk in deze PR even naar de wijzigingen die je doet + de plekken en noem die kort in de README.
Dan is het een soort checklist voor als je het de volgende keer voor een ander item moet doen.